R/render.R

```diff
@@ -391,8 +391,10 @@ render <- function(input,
   perf_timer_start("pandoc")
 
   # compile Rmd to tex when we need to generate --bibliography
-  if (grepl('[.](pdf|tex)$', output_file) &&
-      ('--bibliography' %in% output_format$pandoc$args)) {
+  # or when keep_tex is TRUE
```

Comment:
This commit does not appear to be necessary to me. I don't see what problem you are trying to solve here. When `keep_tex = TRUE`, you will get get a .tex file.